wiki/Storyboard/Scripting/Audio/de.md

```diff
@@ -1,3 +1,8 @@
+---
+outdated: true
```

Comment:
These conflict with #7397 . Should this be changed here?

Reply:
it depends on which pr gets merged first, so leave it as is for now